### PR TITLE
test: BitmapIcon.Foreground effect with ShowAsMonoChrome

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): #14483

## PR Type

What kind of change does this PR introduce?

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 00df175</samp>

Added and improved tests for `BitmapIcon` control. Refactored `Given_BitmapIcon.cs` file for better readability.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs [hunk 1]
```diff
@@ -46,4 +53,46 @@ public async Task When_Themed_Fluent()
...
+			Width = 50,
+			Height = 50,
+			ShowAsMonochrome = false,
+			UriSource = new System.Uri("ms-appx:///Assets/Icons/search.png"),
+			Foreground = new SolidColorBrush(Colors.Green)
+		};
+
+		TestServices.WindowHelper.WindowContent = bitmapIcon;
+		await TestServices.WindowHelper.WaitForLoaded(bitmapIcon);
+		await TestServices.WindowHelper.WaitForIdle();
```
Comment (Youssef1313) [inline src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs:71]: The 3 lines can be replaced by `UITestHelper.Load(bitmapIcon)` I think

Comment (ramezgerges) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs:71]: That one is new to me. Most tests use the longer version anyway, so let's leave it at that.

Comment (jeromelaban) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs:71]: If there's a helper that does it better, let's use it to slowly migrate to it the newer tests :)

Comment (ramezgerges) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs:71]: Sure, but I would tread with caution here. The helper explictly calls `UpdateLayout`, which would cover up layouting and invalidation problems.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**